kernelci/runtime/docker.py

```diff
@@ -32,7 +32,9 @@ def __init__(self, *args, **kwargs):
     def _load_env(self):
         if self.config.env_file and os.path.isfile(self.config.env_file):
             with open(self.config.env_file, encoding='utf-8') as env:
-                return [line.strip() for line in env.readlines()]
+                # skip # and empty lines
+                return [line.strip() for line in env.readlines()
+                        if not line.startswith('#') and line.strip()]
```

Comment:
This is calling `line.strip()` twice, it's OK but a bit sub-optimal.  A neater implementation would be two nested list comprehensions to iterate over the stripped lines.  Not a big deal though, it's arguably easier to read the way it's currently implemented.